Python/Product/PythonTools/PythonTools/Project/CondaEnvCreateInfoBar.cs

```diff
@@ -81,7 +94,18 @@ public CondaEnvCreateInfoBar(PythonProjectNode projectNode)
                         Action = CondaEnvCreateInfoBarActions.Create,
```

Comment:
The CheckAsync function is unusually big. Please split it into smaller chunks for easier readability. For example, create the "Action create" and "Action projectIgnore" in a different function and pass them back.